### ISSUE TITLE
The color of the texts in lists becomes red in PDF output

### ISSUE THREAD

**Issue by Michael-Zhang-07** [aCluelessDanny/typora-theme-ursine#5]
Hi,
I found another small issue, in typora, the color is good, but I am not sure why the color of the texts all become red in the pdf output, like this: :smile:
![image](https://user-images.githubusercontent.com/38196911/54580303-d7e1a480-49dd-11e9-8b01-212f7a2b20ce.png)

Thank you!!!

**Comment by zolrath** [comment]
I am also seeing this behavior running Typora 0.9.68(beta) on Windows 10.

**Comment by aCluelessDanny** [comment]
Sorry about the late response, college really takes a toll on you in the last month or so.

It seems like when it exports to a PDF (or an HTML file), it puts back the regular way how list items are rendered. In Typora, lists are organized like this:
```html
<ul>
	<li>
		<p>Item 1</p>
	</li>
	<li>
		<p>Item 2</p>
	</li>
</ul>
```
Since there's no real way to style the bullet/numbering directly, this would let me do a workaround (albeit a tad hacky), by styling the list (`<li>`) with the red/blue color first, & then subsequently style anything in the list item (in this case, `<p>`) with the text color.

But when exported, it turns into this:
```html
<ul>
	<li>Item 1</li>
	<li>Item 2</li>
</ul>
```
So now the workaround won't work since it will only take the list styling *and not the item styling*.

I could replace the bullet and numbering with psuedo-elements like before, and while this would work alright for bullet lists, it'll bring back the same problem from #4 for numbered lists. So in the end, I think the best solution to this is to set the color of the whole item (bullet and text) to the text color. I'll fix that into the next release if that's alright.

*TL;DR: Lists exported from Typora bypasses the workaround to color bullets and numbering, so the best option is just make the whole item the same color as the text.*
